CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Change Log
 
+## 4.2.2 (April 16 2025)
```

Comment:
Let's make this a minor version bump to `4.3.0` since it's a feature addition. And please also modify the version in `package.json`.

Reply:
Done